src/CMakeLists.txt

```diff
@@ -330,6 +332,13 @@ set (PY_SUPPORT_FILES
 )
 
 #
+# Add the pycapnp Python package to includes
```

Comment:
@rcrowder - Is this going to be an issues with Windows effort? Seems like it should be fine if we are using gcc, no?